### ISSUE TITLE
You can't actually see the name of the music playing

### ISSUE THREAD

**Issue by jahodovykolac** [yairm210/Unciv#5438]
**Platform**
Android 5.1.1

**Version**
Google play, 3.17.4

**Describe the bug**
You can only see what's the playing music when you're opening settings from main menu.

**To Reproduce**
1. Open the 3 lines menu (the one in the upper left corner)
2. The music stops playing
3. When you open settings - you can't see the music, because no music is playing at the moment

**Expected behavior**
The music doesn't stop or it still shows the name of the last played track

**Comment by SomeTroglodyte** [comment]
Interesting. Shouldn't happen as the music is paused, not stopped... Will check once I have repaired my dev hardware.

**Comment by SomeTroglodyte** [comment]
Took me a few days to verify - yes, true. Not quite as I envisioned, too. The easy fix should be just a change in the status check of `MusicController.currentlyPlaying` - when I get around to it.

Pause on menu was a choice - I thought some easily accessible way to take a phone call undisturbed or similar reasons to pause should exist, so I grafted that onto the menu. If you have a better idea...?
